pkgs/development/libraries/linenoise/default.nix

```diff
@@ -0,0 +1,29 @@
+  { stdenv, fetchgit }:
+
+  stdenv.mkDerivation rec {
+    name = "linenoise-${version}";
+    version = "git-2016-09-30";
+
+    src = fetchgit {
+      url = "https://github.com/antirez/linenoise.git";
+      rev = "c894b9e59f02203dbe4e2be657572cf88c4230c3";
+      sha256 = "0wasql7ph5g473zxhc2z47z3pjp42q0dsn4gpijwzbxawid71b4w";
+    };
+
+    phases = ["unpackPhase" "buildPhase" "installPhase"];
```

Comment:
Ideally, you'd only disable phases if they actually fail.

The problem with explicitly listing phases, in general, is that you can miss out on useful phases. This can cause issues with massive closure bloat or run-time crashes. Again, this is more of a general point, I'm sure it won't make that much of a difference in this specific case.